injected/src/features/click-to-load.js

```diff
@@ -1887,7 +1887,7 @@ export default class ClickToLoad extends ContentFeature {
         })
         // Listen to message from Platform letting CTL know that we're ready to
         // replace elements in the page
-        // eslint-disable-next-line promise/prefer-await-to-then
```

Comment:
Can we make sure we use `--report-unused-disable-directives`? Looks like we nolonger needed this.

Reply:
Most of the changes in this PR were a result of `eslint . --fix`. This and other `eslint-disable` lines were removed because they were no longer disabling any errors. I reviewed all occurrences, and they were all correct.

For example, a bunch of "unused variables" are not flagged anymore because we [allow unused function arguments](https://github.com/duckduckgo/eslint-config/blob/main/index.js#L148). That rule came [directly from Standard](https://github.com/standard/eslint-config-standard/blob/v17.1.0/.eslintrc.json#L176), so I suspect we were not applying those rules correctly before.